ecowitt2mqtt/const.py

```diff
@@ -117,3 +118,9 @@
 # Unit systems:
 UNIT_SYSTEM_IMPERIAL = "imperial"
 UNIT_SYSTEM_METRIC = "metric"
+
+# device lookup table:
+DEVICE_DATA = {
```

Comment:
I would:

1. Move this lookup `dict` into your newly created `device.py`.
2. Have the constant merely be a mapping of strings to tuples.
3. Have a helper method that returns a device (returning a "default device" if their particular one isn't in our lookup table).
4. Per my previous comment, if the default device is returned, log a message encouraging the user to let us know about their device.

So, your `device.py` might look like this:

```python
from typing import NamedTuple

from ecowitt2mqtt.const import LOGGER

DEFAULT_DEVICE_MANUFACTURER = "Ecowitt"
DEFAULT_DEVICE_NAME = "Default Ecowitt Device"

DEVICE_DATA = {
    "default": ("Ecowitt", "Default Ecowitt Device"),
    "WH2650": ("Waldbeck", "Waldbeck Hally Weather Station"),
}


class Device(NamedTuple):
    """Simple data object to provide device details."""

    manufacturer: str
    name: str


def get_device_from_model(model: str) -> Device:
    """Return a device based upon a model string."""
    try:
        manufacturer, name = DEVICE_DATA[model]
    except KeyError:
        LOGGER.info(
            "Unknown device model; please report it to us! ",
            "https://github.com/bachya/ecowitt2mqtt",
        )
        manufacturer = DEFAULT_DEVICE_MANUFACTURER
        name = DEFAULT_DEVICE_NAME

    return Device(manufacturer, name)
```